contrib/debian/postrm

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
How did you generate this file?

Reply:
I took the [postrm file in edgelet](https://github.com/Azure/iotedge/blob/master/edgelet/contrib/debian/postrm) and modified that.

Reply:
Generate it properly with `dh_make` please. iotedge uses an old package format, but more importantly your version is a) using the wrong shebang, and b) missing the `DEBHELPER` comment that the builder uses to substitute additional commands.

Reply:
I regenerated the file with `dh_make`. Note that I kept the shebang `#!/bin/bash` because this script uses arrays.

Reply:
Okay, I assume you made it bash because you want to use an array for deleting the users. Just delete them one at a time instead.

Reply:
Fixed. Is there a reason that the preinst is using bash in its shebang?

Reply:
Huh.

Just to be sure, `dh_make` generated the template to use `sh` right? If so, I guess I changed it, but I'm not sure why I did that. It shouldn't have needed to be changed.

Reply:
Yeah, the template (preinst.ex) has `sh`.

Reply:
Yeah, I just verified myself too. Can you change that to `sh` too please?

Reply:
Fixed